app/views/admin/worldwide_organisations_translations/index.html.erb

```diff
@@ -69,12 +69,12 @@
         <%= render "govuk_publishing_components/components/button", {
           text: "Next",
           margin_bottom: 6,
-          # data_attributes: {
```

Comment:
As per previous comment, it would be good to remove this commented text from the commits.